error.go

```diff
@@ -0,0 +1,52 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+package quicktest
+
+import "fmt"
+
+// BadCheckf returns an error used to report a problem with the checker
+// invocation or testing execution itself (like wrong number or type of
+// arguments) rather than a real Check or Assert failure.
+func BadCheckf(format string, a ...interface{}) error {
+	e := badCheck(fmt.Sprintf(format, a...))
+	return &e
+}
+
+// IsBadCheck reports whether the given error is an bad test problem.
```

Comment:
IsBadCheck reports whether the given error has been created by BadCheckf ?

Reply:
Ok.